src/wazuh_modules/syscollector/syscollector_windows.c

```diff
@@ -1123,49 +1123,93 @@ void sys_hw_windows(const char* LOCATION){
 
     /* Call get_baseboard_serial function through syscollector DLL */
     char *serial = NULL;
-    CallFunc2 _get_baseboard_serial;
-    HMODULE sys_library = LoadLibrary("syscollector_win_ext.dll");
-    if (sys_library == NULL)
-    {
-        DWORD error = GetLastError();
-        LPSTR messageBuffer = NULL;
-        LPSTR end;
-        
-        FormatMessage(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS, NULL, error, 0, (LPTSTR) &messageBuffer, 0, NULL);
-        
-        if (end = strchr(messageBuffer, '\r'), end) *end = '\0';
-        
-        mterror(WM_SYS_LOGTAG, "At sys_hw_windows(): Unable to load syscollector_win_ext.dll: %s (%lu).", messageBuffer, error);
-        LocalFree(messageBuffer);
-        
-        serial = strdup("unknown");
-    } else {
-        _get_baseboard_serial = (CallFunc2)GetProcAddress(sys_library, "get_baseboard_serial");
-        if (!_get_baseboard_serial) {
-            mterror(WM_SYS_LOGTAG, "At sys_hw_windows(): Unable to access 'get_baseboard_serial' on syscollector_win_ext.dll.");
+    
+    if(strncmp(getuname(),"Microsoft Windows XP Professional",33)){
```

Comment:
We have the same issue with all Windows versions minor than 6, including Server 2003, Windows 2000, etc. Let's use the function `isVista()` instead.